idaes/generic_models/properties/helmholtz/helmholtz.py

```diff
@@ -58,100 +58,177 @@
-    "func_vfs": {"fname": "vfs",
-                 "units": pyunits.dimensionless,
-                 "arg_units": [pyunits.kJ/pyunits.kg/pyunits.K, pyunits.kPa]},
-    "func_vfu": {"fname": "vfu",
-                 "units": pyunits.dimensionless,
-                 "arg_units": [pyunits.kJ/pyunits.kg, pyunits.kPa]},
-    "func_g": {"fname": "g",
-               "units": pyunits.kJ/pyunits.kg,
-               "arg_units": [pyunits.dimensionless, pyunits.dimensionless]},
-    "func_f": {"fname": "f",
-               "units": pyunits.kJ/pyunits.kg,
-               "arg_units": [pyunits.dimensionless, pyunits.dimensionless]},
-    "func_cv": {"fname": "cv",
-                "units": pyunits.kJ/pyunits.kg/pyunits.K,
-                "arg_units": [pyunits.dimensionless, pyunits.dimensionless]},
-    "func_cp": {"fname": "cp",
-                "units": pyunits.kJ/pyunits.kg/pyunits.K,
-                "arg_units": [pyunits.dimensionless, pyunits.dimensionless]},
-    "func_w": {"fname": "w",
-               "units": pyunits.m/pyunits.s,
-               "arg_units": [pyunits.dimensionless, pyunits.dimensionless]},
-    "func_delta_liq": {"fname": "delta_liq",
-                       "units": pyunits.dimensionless,
-                       "arg_units": [pyunits.kPa, None]},
-    "func_delta_vap": {"fname": "delta_vap",
-                       "units": pyunits.dimensionless,
-                       "arg_units": [pyunits.kPa, None]},
-    "func_delta_sat_l": {"fname": "delta_sat_l", "units": None, "arg_units": [None, None]},
-    "func_delta_sat_v": {"fname": "delta_sat_v", "units": None, "arg_units": [None, None]},
-    "func_p_sat": {"fname": "p_sat",
-                   "units": pyunits.kPa,
-                   "arg_units": [pyunits.dimensionless]},
-    "func_tau_sat": {"fname": "tau_sat",
-                     "units": pyunits.dimensionless,
-                     "arg_units": [pyunits.kPa]},
-    "func_phi0": {"fname": "phi0", "units": None, "arg_units": [None, None]},
-    "func_phi0_delta": {"fname": "phi0_delta", "units": None, "arg_units": [None, None]},
-    "func_phi0_delta2": {"fname": "phi0_delta2", "units": None, "arg_units": [None, None]},
-    "func_phi0_tau": {"fname": "phi0_tau", "units": None, "arg_units": [None, None]},
-    "func_phi0_tau2": {"fname": "phi0_tau2", "units": None, "arg_units": [None, None]},
-    "func_phir": {"fname": "phir", "units": None, "arg_units": [None, None]},
-    "func_phir_delta": {"fname": "phir_delta", "units": None, "arg_units": [None, None]},
-    "func_phir_delta2": {"fname": "phir_delta2", "units": None, "arg_units": [None, None]},
-    "func_phir_tau": {"fname": "phir_tau", "units": None, "arg_units": [None, None]},
-    "func_phir_tau2": {"fname": "phir_tau2", "units": None, "arg_units": [None, None]},
-    "func_phir_delta_tau": {"fname": "phir_delta_tau", "units": None, "arg_units": [None, None]},
+    "func_p": { # pressure as a function of delta and tau
```

Comment:
I am going to assume that the only changes here were adding the comments explaining what each method does. GitHub marked this entire section as changes with no further detail of what the differences were.

Reply:
Some of the external functions were missing units, so I filled in a few things, but nothing much changed.  I think the slight reformatting marked the whole thing changed.